src/site/content/en/blog/payment-and-address-form-best-practices/index.md

```diff
@@ -0,0 +1,782 @@
+language.
+
+```text
+    ZIP code: US
+ Postal code: Canada
+    Postcode: UK
+     Eircode: Ireland
+         PIN: India
+```
+
+It can be irritating or puzzling to be presented with a form that doesn't fit your address or that 
+doesn't use the words you expect. 
+
+Customizing address forms [for multiple locales](https://www.smashingmagazine.com/2020/11/internationalization-localization-static-sites#determining-user-s-language-and-region) may be necessary for your site, but using techniques 
+to maximize form flexibility (as described above) may be adequate. If you don't localize your address 
+forms, make sure to understand the key priorities to cope with a range of address formats:
+* Avoid being over-specific about address parts, such as insisting on a street name or house number.
+* Where possible avoid making fields `required`. For example, addresses in many countries don't 
+have a postal code, and rural addresses may not have a street or road name.
+* Use inclusive naming: 'Country/region' not 'Country'; 'ZIP/postal code' not 'ZIP'.
+
+Keep it flexible!
+
+{% Aside %}
+The [simple address form example above](#address-textarea) can be adapted to work 'well enough' 
+for many locales.
+{% endAside %}
+
+#### Consider avoiding postal code address lookup {: #postal-code-address-lookup}
+
+Some websites use a service to look up addresses based on postal code or ZIP. This may be sensible 
+for some use cases, but you should be aware of the potential downsides.
+
+Postal code address suggestion doesn't work for all countries—and in some regions, post codes can 
+include a large number of potential addresses. 
+
+<figure class="w-figure">
+   <video controls autoplay loop muted class="w-screenshot">
+     <source src="https://samdutton.com/long-list-of-addresses.mp4" type="video/mp4">
+   </video>
+  <figcaption class="w-figcaption">ZIP or postal codes may include a lot of addresses!</figcaption>
+</figure>
+
+It's difficult for users to select from a long list of addresses—especially on mobile if they're 
+rushed or stressed. It can be easier and less error prone to let users take advantage of autofill, 
+and enter their complete address filled with a single tap or click.
+
+<figure class="w-figure">
+   <video controls autoplay loop muted class="w-screenshot">
+     <source src="https://samdutton.com/full-name-autofill.mp4" type="video/mp4">
+   </video>
+  <figcaption class="w-figcaption">A single name input enables one-tap/one-click address entry.</figcaption>
+</figure>
+
+* Use [appropriate payment card autocomplete values](#payment-form-autocomplete).
+* Use a [single input for payment card numbers](#single-number-input).
+* [Avoid using custom elements](#avoid-custom-elements) that break the autofill experience.
+* [Test in the field as well as the lab](#analytics-rum): page analytics, interaction analytics, and 
+real-user performance measurement.
+* [Test across browsers, devices and platforms](#test-platforms). 
```

Comment:
Is this duplication of the checklist items here on purpose?

Reply:
Oops... no! Fixed.